docs/en/ingest-management/index.asciidoc

```diff
@@ -2,6 +2,7 @@
 :beats-repo-dir: {beats-root}
 :im-forum: https://discuss.elastic.co/tags/c/elastic-stack/81/stack-ingest-management
 :code-path: {beats-repo-dir}/libbeat/docs/tab-widgets/code
+:elastic-endpoint-integration: Elastic Endpoint Security
```

Comment:
@dedemorton can we update this attribute to take out  `Elastic` so that it's just `Endpoint Security`? Sorry if this was confusing. See [this comment.](https://github.com/elastic/security-docs/issues/246#issue-701126150)